src/frontend/Komponenter/Behandling/Henleggelse/HenleggModal.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FC, useState } from 'react';
+import { useBehandling } from '../../../App/context/BehandlingContext';
+import Modal from 'nav-frontend-modal';
+import styled from 'styled-components';
+import { Ressurs, RessursStatus } from '../../../App/typer/ressurs';
+import { Radio } from 'nav-frontend-skjema';
+import { FamilieRadioGruppe } from '@navikt/familie-form-elements';
+import { Behandling } from '../../../App/typer/fagsak';
+import { useApp } from '../../../App/context/AppContext';
+import { useNavigate } from 'react-router-dom';
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
```

Comment:
Det skal heller ikke importeres noe fra `nav-frontend-...` forøvrig da alle disse komponentene er utdaterte fra gammelt designsystem. Så komponenter som `Hovedknapp`, `Knapp` og `AlertstripeFeil` skal ikke brukes. Du må finne lignende komponenter fra det nye designsystemet:
https://aksel.nav.no/